code/modules/surgery/organs/tongue.dm

```diff
@@ -92,6 +92,54 @@
 		message = lizard_eckS.Replace(message, "ECKS$1")
 	speech_args[SPEECH_MESSAGE] = message
 
+/obj/item/organ/tongue/lizard/silver
```

Comment:
I think these tongues might be able to be generated by meaty ores.

I personally think that that's okay, but I figured that I should at least tell you about that.

Reply:
also, because they're tongue, you could probably farm for these by giving a silver scale the HARS mutatioin, curing it, healing them up, and then repeating the process.

you'd still have to do the tongue replacement surgeries, of course, so I think that's fine, but again, I thought that you should be made aware of this.